### PR TITLE
Would like to offer this. Closes #48

### LINKED ISSUE #48: Migrate Smite command
Due to the main TFM's reluctance to make the messages less gender specific, I want it re-created in here because I dont agree with being gender specific.

### PR DESCRIPTION
**Pull request by GotNoFriends**

### CODE REVIEW

**Comment by Wild1145** [comment]
The code looks good. It will need to be reviewed, tested and all that, but Im placing it on hold as its not for the current milestone.

**Comment by Wild1145** [comment]
With the recent changes to command handling this will need to be modified to utilize the new command handling system.

**Comment by Wild1145** [comment]
We are now working on the 1.1 Release. Please review any comments and make appropriate modifications.

@GotNoFriends

File: src/com/superiornetworks/icarus/commands/Command_smite.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import com.superiornetworks.icarus.ICM_Utils;
+
+public class Command_smite extends BukkitCommand 
+{
+  @Override
+  public boolean run(final CommandSender sender, Command cmd, String label, String[] args)
+  {
+    if (!TFM_AdminList.isSuperAdmin(sender, true))
```
Comment (Hockeyfan360) [inline src/com/superiornetworks/icarus/commands/Command_smite.java:16]: Use the isRankOrHigher feature, please.

File: src/com/superiornetworks/icarus/commands/Command_smite.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          player.setHealth(0.0);
+          player.setFireTicks(10000);
+          sender.sendMessage(ChatColor.GRAY + "Smitten " + player.getName() + ".");
+          player.sendMessage(ChatColor.DARK_RED + "You were smitten by " + sender.getName() + ".");
+          playerFound = true;
+          break;
+        }
+      }
+      
+      if (playerFound == false)
```
Comment (Hockeyfan360) [inline src/com/superiornetworks/icarus/commands/Command_smite.java:55]: A null-pointer would have already been thrown. Please use the method suggested above.

File: src/com/superiornetworks/icarus/commands/Command_smite.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      {
+        sender.sendMessage(ChatColor.GRAY + args[0] + " not found!");
```
Comment (Hockeyfan360) [inline src/com/superiornetworks/icarus/commands/Command_smite.java]: Needs to return true, after being moved to the top.

File: src/com/superiornetworks/icarus/commands/Command_smite.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    
+    if (length == 0)
+    {
+      sender.sendMessage("/smite <player>");
+      return true;
+    }
+    
+    if (length == 1)
+    {
+      boolean playerFound = false;
```
Comment (Hockeyfan360) [inline src/com/superiornetworks/icarus/commands/Command_smite.java:32]: A boolean is not needed, test to see if the player is null, return true, and be done.

File: src/com/superiornetworks/icarus/commands/Command_smite.java [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+        }
+    
+    int length = args.length;
+    
+    if (length == 0)
+    {
+      sender.sendMessage("/smite <player>");
+      return true;
+    }
+
```
Comment (Hockeyfan360) [inline src/com/superiornetworks/icarus/commands/Command_smite.java:29]: You need to make sure that the player you are smiting is actually a player.

**Comment by Wild1145** [comment]
@GotNoFriends Any update on this PR?

**Comment by Wild1145** [comment]
Closed for inactivity.

**Pull request closed without merging**